pkg/gardenlet/operation/botanist/kubecontrollermanager_test.go

```diff
@@ -10,6 +10,7 @@ import (
 	"net"
 
 	"github.com/Masterminds/semver/v3"
+	dwdapi "github.com/gardener/dependency-watchdog/api/prober"
```

Comment:
So far this import was imported as
```golang
	proberapi "github.com/gardener/dependency-watchdog/api/prober"
```

You could keep it consistent with the existing codebase. 

```suggestion
	proberapi "github.com/gardener/dependency-watchdog/api/prober"
```

You could also add import alias rule for this import in https://github.com/gardener/gardener/blob/df16d9b9a5b4603fe5f6f495bad260fc4f892688/.golangci.yaml.in#L15-L17.

Reply:
Addressed with 885417aa639c7aacb76626a17f5a6154c4039c96

Reply:
I pushed https://github.com/gardener/gardener/pull/12272/commits/d582641b140f7310479ae3aa839306a6d922b3fa to add the suggested import alias rules.